server/src/main/java/io/kroki/server/Server.java

```diff
@@ -68,7 +69,13 @@ public void start(Promise<Void> startPromise) {
   }
 
   static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer>> listenHandler) {
-    HttpServer server = vertx.createHttpServer();
+    Integer maxInitialLineLength = config.getInteger("KROKI_MAX_URI_LENGTH");
+    HttpServerOptions serverOptions = new HttpServerOptions();
+    if (maxInitialLineLength != null) {
+      serverOptions.setMaxInitialLineLength(maxInitialLineLength);
+    }
+
+    HttpServer server = vertx.createHttpServer(serverOptions);
```

Comment:
I've renamed the variable name to `maxUriLength`:

```suggestion
    Integer maxUriLength = config.getInteger("KROKI_MAX_URI_LENGTH");
    HttpServerOptions serverOptions = new HttpServerOptions();
    if (maxUriLength != null) {
      serverOptions.setMaxInitialLineLength(maxUriLength);
    }
    HttpServer server = vertx.createHttpServer(serverOptions);
```